ISISScientificComputing/autoreduce
# Add tests for end of run monitor

#### Changes
Add unit tests for the `end_of_run_monitor` or as much as can be tested in the linux env that we run tests in. 
* Some minor changes have been made to the  end of run monitor these are mainly dealing with swapping to a settings file and avoiding hard coded (*with slashes*) in favour of `os.path.join(...)` so we no longer need to deal with `\\` vs `/`
* minor change to `data_archive_creator` to ensure that cycle directories are made in both the `data` and `logs` directories

#### To Test
* Travis must pass all tests and we need an acceptable test coverage for end of run monitor
* Put this end of run monitor on the development server and ensure that it still picks up files

Fixes #194
